atd-vzd/migrations/default/1715960018005_indexes/up.sql

```diff
@@ -1,8 +1,9 @@
 create index on public.people_cris (unit_id);
 create index on public.people_cris (prsn_injry_sev_id);
-create index on public.people_cris (crash_id);
-create index on public.people_cris (crash_id, unit_nbr, prsn_nbr); -- charges -> person trigger
-create index on public.units_cris (crash_id, unit_nbr); -- people -> unit_id trigger
+create index on public.people_cris (cris_crash_id);
+create index on public.people_cris (cris_crash_id, unit_nbr, prsn_nbr); -- charges -> person trigger
+create index on public.people_cris (unit_id); -- charges -> crash id trigger
```

Comment:
small thing but this index looks like a duplicate of line 1 in this migration

Reply:
great catch! i believe postgres really will create two identical index for you 💣